test/save_test.rb

```diff
@@ -69,11 +78,21 @@ def test_touch_will_expire_the_caches
     @record.touch
   end
 
-  def test_expire_cache_works_in_a_transaction
+  def test_expire_cache_works_in_a_transaction_when_forced
     expect_cache_delete("#{NAMESPACE}attr:Item:id:id/title:#{cache_hash('"1"/"bob"')}")
     expect_cache_delete("#{NAMESPACE}attr:Item:id:title:#{cache_hash('"bob"')}")
     expect_cache_delete(@blob_key)
 
+    ActiveRecord::Base.transaction do
+      @record.expire_cache(force: true)
```

Comment:
The fact that this test needs to be updated in this way shows the breaking change I mentioned in the change to `expire_cache`